ios/OpenTokReactNative/OTSessionManager.swift

```diff
@@ -399,12 +399,25 @@ class OTSessionManager: RCTEventEmitter {
         resolve(supportedCodecs)
     }
     
-    @objc func forceMuteAll(_ sessionId: String, excludedStreamIds: NSArray, resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) -> Void{
+    @objc func forceMuteAll(_ sessionId: String, excludedStreamIds: Array<String>, resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) -> Void{
         guard let session = OTRN.sharedState.sessions[sessionId] else {
             reject("event_failure", "Session ID not found", nil)
             return
         }
-        return resolve(true);
+        var excludedStreams:[OTStream] = []
+        for streamId in excludedStreamIds {
+            guard let stream = OTRN.sharedState.subscriberStreams[streamId] ?? OTRN.sharedState.publisherStreams[streamId] else {
```

Comment:
Do we need to check for subscriber & publisher streams?

Reply:
Yes. You can exclude any stream from being muted (and excluding your own published stream is a common practice).

Reply:
Yes, that's correct. But my point is, on a normal scenario the only publisher stream is the current publisher (the user itself), we should not mute the current user who is performing the mute-all action. I think the `excludedStreamIds` should be used only for subscribers. WDYT?

Reply:
The native SDK API does not restrict which sorts of streams you can mute (those published by your client or others). The API in the React Native SDK should mirror the behavior in the native SDKs.

Reply:
Ok, so maybe we need some docstrings with an explanation about it?

Reply:
@jvalli it's already in the docs. See the docs for `OTSession.forceMuteAll()` here: `https://tokbox.com/developer/sdks/react-native/reference/OTSession.html`.